swephp.c

```diff
@@ -2736,6 +2942,43 @@ PHP_FUNCTION(swe_lun_occult_when_loc)
 	}
 }
 
+/* {{{ pod
+=pod
+
+=head1 function swe_sol_eclipse_when_glob(tjd_start, iflag, ifltype, backw);
+
+When is the next solar eclipse anywhere on earth?
+
+=head3 Parameters
+
+  tjd_start    double      Julian day number, Universal Time
+  iflag     int         (specify ephemeris to be used, cf. swe_calc( ))
+  ifltype   int         Eclipse type to be searched; 0 if any type of eclipse is wanted
+  backw     int         search backward in time
+
+=head3 return array
+
+      retflag => (int)            ERR or eclipse type
+		returns SE_ECL_TOTAL or SE_ECL_ANNULAR or SE_ECL_PARTIAL or SE_ECL_ANNULAR_TOTAL
+				SE_ECL_CENTRAL
+				SE_ECL_NONCENTRAL
+      serr    => (string)         Error string, on error only
+      tret    => array of 8 double:
+        tret[0]	time of maximum eclipse
+        tret[1]	time, when eclipse takes place at local apparent noon
+		tret[2] time of eclipse begin
+		tret[3] time of eclipse end
+		tret[4] time of totality begin
+		tret[5] time of totality end
+		tret[6] time of center line begin
+		tret[7] time of center line end
```

Comment:
```suggestion
		tret[7] time of center line end
		tret[8] Unused/not implemented.
		tret[9] Unused/not implemented.
```
Since array indexes 8 & 9 are returned, I think it's valuable to document this for the user.